oak-store-spi/pom.xml

```diff
@@ -26,7 +26,11 @@
             <instructions>
               <Export-Package>
                 org.apache.jackrabbit.oak.jcr,
-                org.apache.jackrabbit.oak.jcr.observation.filter
+                org.apache.jackrabbit.oak.jcr.observation.filter,
+                org.apache.jackrabbit.oak.json,
+                org.apache.jackrabbit.oak.spi.cluster,
+                org.apache.jackrabbit.oak.spi.commit,
+                org.apache.jackrabbit.oak.spi.state
```

Comment:
checking again with oak-core i guess we actually had everything exported. so you should be fine removing the Export-Package